File: src/main/java/com/innercircle/survey/repository/ResponseRepository.java
package com.innercircle.survey.repository;

import com.innercircle.survey.model.Response;
import com.innercircle.survey.model.Survey;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface ResponseRepository extends JpaRepository<Response, Long> {
    List<Response> findBySurveyAndVersion(Survey survey, Integer version);
}

File: survey-domain/src/main/kotlin/com/innercircle/surveydomain/repository/ResponseRepository.kt
package com.innercircle.surveydomain.repository

import com.innercircle.surveydomain.model.Response
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

// 어노테이션은 주석으로 명시하는 역할 24.07.27
@Repository
interface ResponseRepository : JpaRepository<Response, Int>
